src/components/DigitButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+type Props = {
+  digit: number;
+  onClickDigitBtn: (digit: number) => (event: React.MouseEvent<HTMLButtonElement>) => void;
+};
+
+function DigitButton({ digit, onClickDigitBtn }: Props) {
+  return (
+    <button className="digit" type="button" onClick={e => onClickDigitBtn(digit)(e)}>
```

Comment:
```suggestion
    <button className="digit" type="button" onClick={onClickDigitBtn(digit)}>
```
onClickDigitBtn(digit)에 event를 파라미터로 받는 함수를 반환하고 있어서 이렇게 변경해도 동일해요 🙂